README.md

```diff
@@ -148,129 +160,160 @@ The data source for the LiDAR data is [opentopography]( https://portal.opentopog
 
-## BoundaryConditions data
+<br>
 
-### Rain gauges location
+## Dynamic Boundary Conditions
 
-The rain gauges location is accessed from [HIRDS](https://hirds.niwa.co.nz/) which is tool that provides a map-based interface to enable rainfall estimates to be provided at any location in New Zealand. The gauges information can be stored in the database using `hirds_gauges.py` script as shown below:
+### Rainfall sites' locations
+
+The rainfall sites' locations are accessed from [NIWA HIRDS](https://hirds.niwa.co.nz/) which is a tool that provides a map-based interface to enable rainfall estimates to be provided at any location in New Zealand. The sites' information can be stored in the database using the `rainfall_sites.py` script as shown below:
 
 ```python
 #!/usr/bin/env python
 if __name__ == "__main__":
     from src.digitaltwin import setup_environment
+    from src.dynamic_boundary_conditions import rainfall_sites
+    
     engine = setup_environment.get_database()
-    guages = get_hirds_gauges_data()
-    hirds_gauges_to_db(engine, guages)
+    sites = rainfall_sites.get_rainfall_sites_data()
+    rainfall_sites.rainfall_sites_to_db(engine, sites)
 ```
 
-### To get the rainfall data of the gauging sites within the desired catchment area from HIRDS and store in the database, use **hirds_depth_data_to_db.py** script.
+<br>
+
+### Store rainfall data to database
 
-**hirds_depths_to_db(engine, catchment_area, path)** function requires three arguments,
-1. engine: information about the database to interact with.
-2. catchment_area: Polygon type
-3. path: hirds depth data is first downloaded as csv files in the local directory before getting stored in the database, so user needs to provide the path where the depth data will be stored as csv files.
-The example is as shown below:
+To store the rainfall data of sites within the desired catchment area in the database, the `hirds_depth_data_to_db.py` script is used. As shown below:
 
 ```python
 #!/usr/bin/env python
 if __name__ == "__main__":
+    import pathlib
     from src.digitaltwin import setup_environment
+    from src.dynamic_boundary_conditions import hyetograph
+    from src.dynamic_boundary_conditions import hirds_depth_data_to_db
+    
+    catchment_file = pathlib.Path(r"src\dynamic_boundary_conditions\catchment_polygon.shp")
+    file_path_to_store = pathlib.Path(r"U:\Research\FloodRiskResearch\DigitalTwin\hirds_rainfall_data")
     engine = setup_environment.get_database()
-    file = r'P:\Data\catch4.shp'
-    path = r'\\file\Research\FloodRiskResearch\DigitalTwin\hirds_depth_data'
-    catchment = geopandas.read_file(file)
-    catchment = catchment.to_crs(4326)
-    catchment_area = catchment.geometry[0]
-    hirds_depths_to_db(engine, catchment_area, path)
+    catchment_polygon = hyetograph.catchment_area_geometry_info(catchment_file)
```

Comment:
Good to see fixed file paths replaced @caseyli92. Also good to see the REAME being kept up to date! Good work!